src/components/Dashboard/StaffNotation.vue

```diff
@@ -386,13 +381,15 @@ export default class StaffNotation extends Mixins(AllowableActionsMixin, FilingM
 
       if (isNaN(id)) throw new Error('Invalid API response')
 
-      // navigate to Create UI if Full/Limited restoration
-      // navigate to Edit UI if Limited extension/Full to Limited conversion
-      if (applicationName === ApplicationTypes.CREATE_UI) {
-        url = `${this.getCreateUrl}?id=${this.getIdentifier}`
-      }
-      if (applicationName === ApplicationTypes.EDIT_UI) {
-        url = `${this.getEditUrl}${this.getIdentifier}/` + restorationType + `?restoration-id=${id}`
+      // navigate to Edit UI for limited restoration extension filing
+      // navigate to Edit UI for limited restoration to full filing
+      // navigate to Create UI for full or limited restoration filing
+      if (restorationType === FilingSubTypes.LIMITED_RESTORATION_EXTENSION) {
+        url = `${this.getEditUrl}${this.getIdentifier}/limitedRestorationExtension?restoration-id=${id}`
+      } else if (restorationType === FilingSubTypes.LIMITED_RESTORATION_TO_FULL) {
+        url = `${this.getEditUrl}${this.getIdentifier}/limitedRestorationToFull?restoration-id=${id}`
+      } else {
+        url = `${this.getCreateUrl}restoration-business-name?id=${this.getIdentifier}`
```

Comment:
I've added this direct route to step 1 of the restoration filing, which saves a default re-route and then another re-route for restorations.

Ditto in TodoList.

And also I updated the unit tests.

Reply:
@hfekete Here's why a route is specified for Create UI ^^.

Reply:
@patrickpeinanw FYI, and also see other changes in this PR. Thanks!